action.yml

```diff
@@ -17,6 +17,9 @@ inputs:
   title:
     description: The pull request title. Default to `Version Packages`
     required: false
+  customGitUser:
```

Comment:
I would prefer for this to be called something like `setupGitUser` - with the default being `true`.

Reply:
Sounds good, updated